### ISSUE TITLE
DO NOT TRY THIS PROGRAM !!! THIS THEFT ITEMS

### ISSUE THREAD

**Issue by flakka2022** [Gunthersuper/simple-steam-idler-replit#1]
**ATTENTION !!!!**
I USED THIS AND AFTER 15 DAYS ALL ITEMS ON MY INVENTORY SOLD IN STEAM MARKET WITH A CHEAP PRICE AND A USER BUY IT ALL FOR CHEAP . LIKE THEY SOLD MY INVENTORY WICH WORTH OVER 1K$ FOR JUST 10$ AND **SCAMMED** ME IN THIS WAY . BE AWARE OF THIS PROGRAM .

**Comment by blainsage** [comment]
> flakka2022

the code isn't malicious. i just wouldnt push any code to the repo. im sure if you posted your own configuration with your account info, sure he could do something like this.

**Comment by flakka2022** [comment]
> > flakka2022
> 
> the code isn't malicious. i just wouldnt push any code to the repo. im sure if you posted your own configuration with your account info, sure he could do something like this.

yeah cuz of this i try to warn other peoples to be carefull . im sure owner of the code had purpose to do this and made people to use repo and dont warn them there info is public !!!!!!

**Comment by ikmauro** [comment]
so basically it isnt safe?

**Comment by flakka2022** [comment]
NO IS NOT . MIGHT BE RUN IT LOCALY SAFE BUT I DONT RECOMMEND CUZ THEY
SCAMMED ME .

**Comment by BenjiThatFoxGuy** [comment]
looking at the code, it's safe and literally doesn't do anything other than log in to steam and pretend you're playing a certain game.
i'm 100 percent sure this person just cloned this and ran it through a public repo instead of making sure to keep their login details secret and someone took advantage of that.

**Comment by Beachmountainbe** [comment]
This program itself does not steal any credits, If you do not make your username and password private, anyone can login and the login details are out in the open.

**Comment by Beyulium** [comment]
+

**Comment by flakka2022** [comment]
PLEASE READ ALL OF MY COMMENTS UP HERE . I TOLD WHAT HAPPEND TO ME AND HOW . WHEN PUBLISHER SAYS RUN IT ON REPLIT without warning WHERE ALL PRIVATE INFORMATION WILL BE PUBLIC , THIS SHOW PUBLISHER PURPOSE IS NOT GOOD AND THEY ARE INVOLVED IN ITEM THEFT . i just try to warn people who are intrested to execute this code on replit where publisher advice to use .

**Comment by BenjiThatFoxGuy** [comment]
the fact that they didn't state the obvious doesn't make it any less of your fault, let alone you reaching so far and beyond and accusing a dev which made a tool like this.

**Comment by Bossman9090** [comment]
you cant even make the replit private.he should add a way to hide the password and stuff.

**Comment by BenjiThatFoxGuy** [comment]
Maybe you’re right, but this doesn’t mean that it’s ok to jump to accusations this was done on purpose.
I don’t use secrets fields in many of my projects cuz they’re private repos plugged into stuff that allows stuff to be safe and private like CF workers - it’s sometimes a hassle. I think the dev just made it for themselves and just made the repo public because why not let people use it.
If someone’s gonna blindly take code and run it, that’s on them for what happened. Because I sure as hell don’t see any token stealing in it.

**Comment by Bossman9090** [comment]
i didnt accuse him.Just saying theres no way to make it private not evne encrypt the files which is not possible at alll

**Comment by BenjiThatFoxGuy** [comment]
I mean this in relation to the guy that started the issue generally.

**Comment by Bossman9090** [comment]
he probably had no steam guard or so

**Comment by BenjiThatFoxGuy** [comment]
lol yeah
Also it may technically be possible to encrypt, but the code to decrypt it and salt/keys would have to be public as well defeating the purpose.

**Comment by Bossman9090** [comment]
im not an expert in programming so i barely know shit about this.Since heruku method got patched theres no other way to do it than this ig

**Comment by BenjiThatFoxGuy** [comment]
There is, there's like free tiers at cloud providers (e.g. Oracle Cloud). But ya it requires some proper sysadmin knowledge to setup plus a credit card to verify. But ya, it's a good idea to have a host especially if you ever code your own shit.

**Comment by Bossman9090** [comment]
Guess that u gotta pay to farm hours on steam now huh.Back then it was free and easy to do so

**Comment by BenjiThatFoxGuy** [comment]
Nah I literally just said it's free xD

**Comment by flakka2022** [comment]
> he probably had no steam guard or so

i had but it takes steam guard secret key . so they can generate steam guard key easily .
